app/controllers/csv_files_controller.rb

```diff
@@ -19,12 +19,12 @@ def show
   end
 
   def destroy
-    @csv_file = CsvFile.find(params[:id])
     @csv_file.destroy
     redirect_to csv_files_path
   end
 
   def index
+    @csvfiles = current_user.csv_files
```

Comment:
Could you add  an underscore in here so that its csv_files just so it matches the format of the other variables